File: studies/movies/ggplot2/box_faceted.R
library(tidyverse)

df <- read_csv("data/movies.csv")
ggplot(df) +
  geom_boxplot(aes(
    y = `Production Budget`,
    fill = `MPAA Rating`
  )) +
  facet_wrap(vars(`Major Genre`), ncol = 5)

File: galleries/movies/altair/box_faceted.py
import pandas as pd
import altair as alt

df = pd.read_csv("data/movies.csv")

chart = (
    alt.Chart(df)
    .mark_boxplot()
    .encode(
        x="MPAA Rating",
        y="Production Budget",
        color="MPAA Rating",
        facet=alt.Facet("Major Genre").columns(5),
    )
)
